src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  margin: 0;
+}
```

Comment:
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/98764641/197799007-29aa8a99-4f7d-4747-a0de-6678b56d84e4.png)

font props move to `html` tag. it is better for performance